resources/home/dnanexus/tests/test_make_workbook.py

```
@@ -17,56 +17,44 @@
 from types import SimpleNamespace
 
 
-class TestWorkbook():
-    '''
+class TestWorkbook:
+    """
     Tests for excel() class in make_workbook script
-    '''
+    """
+
     summary_content = {}
```

Ruff (0.16.0)

[warning] Mutable default value for class attribute

(RUF012)